django_ai_assistant/langchain/chat_message_histories.py

```diff
@@ -73,13 +77,17 @@ def add_messages(self, messages: Sequence[BaseMessage]) -> None:
             messages: A list of BaseMessage objects to store.
         """
         with transaction.atomic():
+            existing_message_ids = [m.id for m in self.get_messages()]
+
+            messages_to_create = [m for m in messages if m.id not in existing_message_ids]
```

Comment:
@fjsj although this was working I've update it to compare by ID, I think this makes it safer to internal changes